lib/dns.js

```diff
@@ -272,12 +293,12 @@ exports.resolve = function(hostname, type_, callback_) {
   if (typeof resolver === 'function') {
     return resolver(hostname, callback);
   } else {
-    throw new Error('Unknown type "' + type_ + '"');
+    throw new Error(`Unknown type ${type_}`);
   }
 };
 
 
-exports.getServers = function() {
+exports.getServers = () => {
   return cares.getServers();
 };
```

Comment:
I remember the discussion regarding the parens in arrow function parameters, but not sure about the body. I'll have to look. But I feel that having curly braces for even a single expression kinda beats the purpose of arrow functions.